src/constants.ts

```diff
@@ -0,0 +1,80 @@
+export const TOKEN_SYMBOLS_MAP = {
```

Comment:
Should we add a comment that these should all be checksummed?

Reply:
Added here https://github.com/across-protocol/sdk-v2/pull/141/commits/c96a884bc1daf827578340fb87f5e2b36bc5383a#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R12